tests/lib/utils

```diff
@@ -52,3 +52,12 @@ function improver_compare_output {
   [[ "$status" -eq 0 ]]
   [[ "$output" =~ "are identical." ]]
 }
+
+function improver_check_recreate_kgo {
```

Comment:
Do you think that it would be a good idea to print some sort of message, so that someone knows that the RECREATE_BATS_KGO variable is set, and, so files will be copied into this directory? At the moment, I'd be slightly concerned that someone might overwrite known good output that they'd previously created, as they'd forgotten that RECREATE_BATS_KGO was still set. This message could potentially go into `setup()` within `bats-exec-test` or somewhere else.

Reply:
I've added a prompt at the beginning of running the tests with this mode on. Press enter to continue (if you don't want to, I assume people know about CTRL-C). It also details how to turn it off:

```
improver tests cli combine

Set to recreate KGO at path /data/users/bayliffe/NewTests
To unset this use: unset RECREATE_BATS_KGO 

Press enter to continue
```